src/atsphinx/typst/writer.py

```
@@ -130,6 +133,23 @@ def depart_desc_content(self, node: addnodes.desc_content):
         self._hi.pop()
         self.body.append(f"{self._hi.indent}],\n")
 
+    def visit_index(self, node: addnodes.index):
+        # NOTE: This is very simple implementation.
+        #   There may be a more correct implementation.
+        # TODO: Implement other cases.
+        self.packages.add("@preview/in-dexter:0.7.2")
+        self.context["has_index"] = True
+        for entry in node.get("entries", []):
+            entrytype, entryname, target, ignored, key = entry
```

Ruff (0.15.12)

[warning] Unpacked variable `target` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)